recipe/meta.yaml

```
@@ -35,7 +35,9 @@ requirements:
     - conda-standalone
     - pillow >=3.1     # [win or osx]
     - nsis >=3.08      # [win]
+    - pywin32          # [win]
     - jinja2
+    - jsonschema
```

Do we want to constrain the version like the meta.yaml in the constructor repo? 
https://github.com/conda/constructor/blob/74e4e36f182855c1eb21ca5fb1aeba2a0485738e/recipe/meta.yaml#L30C7-L30C21

```suggestion
    - jsonschema >=4
```